internal/verify/validate.go

```diff
@@ -257,6 +257,17 @@ func ValidStringIsJSONOrYAML(v interface{}, k string) (ws []string, errors []err
 	return
 }
 
+// ValidTimeDuration validates a string can be parsed as a valid time.Duration
+func ValidTimeDuration(v interface{}, k string) (ws []string, errors []error) {
+	_, err := time.ParseDuration(v.(string))
+
+	if err != nil {
+		errors = append(errors, fmt.Errorf("%q cannot be parsed as a valid time.Duration: %w", k, err))
```

Comment:
Referencing `time.Duration` leaks an internal implementation detail to practitioners. Maybe something like

```suggestion
		errors = append(errors, fmt.Errorf("%q cannot be parsed as a duration: %w", k, err))
```